src/Kernel-Tests/CompiledBlockTest.class.st

```diff
@@ -13,9 +13,9 @@ CompiledBlockTest >> testLiteralEqual [
 
 	| value |
 	value := 0.
-	10 timesRepeat: [self assert: value equals: 0].
+	(1 to: 10) do: [self assert: value equals: 0].
```

Comment:
```suggestion
	"Do not use #to:do: because it is optimized and will prevent us of testing the behavior we want."
	(1 to: 10) do: [self assert: value equals: 0].
```